lesson05/assignment/index.html

```diff
@@ -8,8 +8,37 @@
         <title>Assignment 05 - CSS Grid</title>
     </head>
     <body>
-        <main class="main">
-          
-        </main>
+      <!--HEADER-->
+      <div class="header">
+        <h1>Header</h1>
+      </div><!--close HEADER div-->
+
+      <!--NAVIGATION-->
+      <nav>
+        <ul class="ul-navigation">
```

Comment:
For implementing BEM here, you can give the `<nav>` a class of `nav` and the `<ul>` a class of `nav__list` and the `<li>`s a class of `nav__item` and the `<a>`s with a class of `nav__link`.